src/main/java/stsjorbsmod/cards/cull/Anxiety.java

```diff
@@ -0,0 +1,62 @@
+package stsjorbsmod.cards.cull;
+
+import com.megacrit.cardcrawl.actions.common.DrawCardAction;
+import com.megacrit.cardcrawl.actions.common.GainEnergyAction;
+import com.megacrit.cardcrawl.actions.defect.IncreaseMiscAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Cull;
+
+public class Anxiety extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(Anxiety.class);
+
+    private static final CardRarity RARITY = CardRarity.RARE;
+    private static final CardTarget TARGET = CardTarget.SELF;
+    private static final CardType TYPE = CardType.SKILL;
+    public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
+
+    private static final int COST = 0;
+    private static final int INITIAL_DRAW = 3;
+    private static final int UPGRADE_DRAW = 1;
+
+    private static final int ENERGY = 1;
+    private static final int DRAW_LOSS = 1;
+
+    public Anxiety() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+        this.baseMetaMagicNumber = metaMagicNumber = INITIAL_DRAW;
+        this.misc = metaMagicNumber;
+        baseMagicNumber = magicNumber = DRAW_LOSS;
+        baseUrMagicNumber = urMagicNumber = ENERGY;
+        this.exhaust = true;
+    }
+
+    @Override
+    public void use(AbstractPlayer p, AbstractMonster abstractMonster) {
+        addToBot(new DrawCardAction(this.misc));
+        addToBot(new GainEnergyAction(this.urMagicNumber));
+        if (this.misc > 0) {
+            this.addToBot(new IncreaseMiscAction(this.uuid, this.misc, -this.magicNumber));
+        }
+    }
+
+    public void applyPowers() {
+        this.baseMetaMagicNumber = this.misc;
```

Comment:
I think you'll want to also override `applyLoadedMiscValue` with this line, to ensure the number will show correctly in the master deck after a save/load transition

Reply:
I'm not a 100% sure if I did the override correctly. I'm calling super.applyLoadedMiscValue and initializeDescription, but I'm not totally sure if that's needed for anything.